### DISCUSSION TITLE
Cloudflare CORS

### DISCUSSION THREAD

**Discussion by ali-master** [vercel/next.js#11247] [Help]
Hi there,

We are using Cloudflare as our DNS manager and we are under a huge DDOS attack and when we do active the Cloudflare firewall on our site, then we receive CORS errors, but when we deactivate the firewall, then everything goes right.

Is there anyone who knows how we can fix this problem?

**Comment by chrisciszak** [comment]
Hey!

Did you manage to fix this?

> Hi there,
> 
> 
> 
> We are using Cloudflare as our DNS manager and we are under a huge DDOS attack and when we do active the Cloudflare firewall on our site, then we receive CORS errors, but when we deactivate the firewall, then everything goes right.
> 
> 
> 
> Is there anyone who knows how we can fix this problem?
